deploy/awstools/awstools.py

```diff
@@ -22,10 +22,16 @@
 from mypy_boto3_ec2.type_defs import FilterTypeDef
 from mypy_boto3_s3.literals import BucketLocationConstraintType
 
-# setup basic config for logging
+
 if __name__ == '__main__':
+    # setup basic config for logging
     logging.basicConfig()
 
+    # use __builtin.input because we aren't in a StreamLogger context
+    from __builtin__ import input as firesim_input
+else:
+    from util.io import firesim_input
```

Comment:
```suggestion
from pathlib import Path
sys.path.append(Path( __file__ ).parent.absolute() + '../')
```

Reply:
Your suggested patch isn't quite right, it would leave `firesim_input` undefined for the uses below in the file.

Reply:
Right... My suggestion should have gone before the import of firesim_input.